src/components/common/Sign/SignForm.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+
+import { StyledSignFormContainer, StyledSignFormTitle, StyledSignFormBox } from './SignForm.styled'
+import { postSignUp, postSignin } from '../../../apis/auth'
+
+type Props = {
```

Comment:
컴포넌트 프롭은 컴포넌트명 + "Props"로 네이밍 컨벤션을 유지하면 좋을 것 같습니다.

그래서

```suggestion
type SignFormProps = {
```
형식을 사용하는 것이 좋아보입니다.

또한, 컴포넌트 프롭은 다른 타입을 `extends`해서 사용하는 경우가 종종 발생하니, 컴포넌트 프롭을 정의할 때는 `interface`로 정의하여 사용하는 것이 좋아보입니다.

네이밍 컨벤션 관련하여 [네이밍 컨벤션Discussion](https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-1-7/discussions/9)를 참고해주시면 감사하겠습니다.

Reply:
https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-1-7/commit/d3b9cbc35126fb1e0a39f910dc758ec7c725eaee

감사합니다. 수정완료했습니다!